protocol/v1/qbft/validation/changeround/validate_justification.go

```diff
@@ -26,6 +27,14 @@ func Validate(share *beacon.Share) pipelines.SignedMessagePipeline {
 	}
 }
 
+// ValidateWithRound is the constructor of validateJustification with round
+func ValidateWithRound(share *beacon.Share, round specqbft.Round) pipelines.SignedMessagePipeline {
```

Comment:
why we need with round? where in spec need it?

Reply:
https://github.com/bloxapp/ssv-spec/blob/75fdec56196df1c1f494fc54f6a10df838c27ce2/qbft/round_change.go#L218 here spec checks the round from the message, not message's justification, so we need to pass it from message